package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "vscoq",
+	"displayName": "VSCoq",
+	"description": "An IDE for the Coq Proof Assistant",
+	"author": {
+		"name": "Christian J. Bell",
+		"email": "siegebell@gmail.com",
+		"url": "http://people.csail.mit.edu/cj/"
+	},
+	"version": "0.2.8",
```

Comment:
The version in `master` is now 0.3.0. It was probably updated after you started working on this PR.